uptane-standard.md

```diff
@@ -273,7 +273,7 @@ The following use cases provide a number of scenarios illustrating the manner in
 An OEM plans to install Uptane on new vehicles. This entails the following components: code to perform full and partial verification, the latest copy of the relevant metadata, the public keys, and an accurate attestation of the latest time. The OEM then either requires its tier-1 suppliers to provide these materials to the suppliers' assembly lines or can choose to add the materials later at the OEM's assembly lines. The OEM's implementation is Uptane-compliant if:
 
 1. all primaries perform full verification;
-1. all secondaries that are updated via OTA perform full or partial verification; and
+1. all secondaries that are updated via OTA perform partial verification at a minimum; and
```

Comment:
A little bit vague, because there's only minimum (partial) and maximum (full).

Reply:
I think this is a question of emphasis @trishankatdatadog. We want to establish partial verification as a required baseline. If you think the word "minimum" is confusing in this context, maybe it should be phrased "all secondaries that  are updated via OTA MUST at least perform partial verification" ..

Reply:
```suggestion
1. all secondaries that are updated via OTA MUST at least perform partial verification; and
```

Reply:
Here again...I think this answers the concern